main/src/main/scala/sbt/internal/Act.scala

```diff
@@ -424,7 +424,7 @@ object Act {
       def evaluate(kvs: Seq[ScopedKey[_]]): Parser[() => State] = {
         val preparedPairs = anyKeyValues(structure, kvs)
         val showConfig = if (action == PrintAction) {
-          Aggregation.ShowConfig(true, true, println, false)
+          Aggregation.ShowConfig(true, true, s => { print(s + "\n"); System.out.flush() }, false)
```

Comment:
How does splitting `println` to `print` + `flush` prevent interleaving? Also `\n` should be replaced with `System.lineSeparator`?

Reply:
It's possible that this is wrong. I went ahead and reverted that commit.